slash/react/src/Table/Pagination/Li.tsx

```diff
@@ -30,7 +31,8 @@ const Li = ({ isVisible, active, value, ...props }: LiProps) => {
       <a
         className="af-pager__item-link"
         href="/#"
-        onClick={onClick({ ...props, value })}
+        aria-label={`Page ${value} des ${pageLinkLabel}`}
```

Comment:
Pareil ici pour le verbe d'action 
```suggestion
        aria-label={`Afficher la page ${value} des ${pageLinkLabel}`}
```

Reply:
Les lecteurs d'écran lisent déjà les rôles, ici il va dire "Lien Page 2 des sinistres", ce qui est préférable à "Lien Afficher la page 1 des sinistres"

Reply:
tu as raison :)